app/helpers/forms_helper.rb

```diff
@@ -122,6 +122,17 @@ def sms_example_for_question(qing)
     end
   end
 
+  def rank_for_sms_question(qing)
```

Comment:
Would the `full_rank` method not work here?

This approach seems kind of unorthodox...